ingest/ingest_pipeline.py

```diff
@@ -413,6 +424,7 @@ def main() -> None:
     parsed_args = create_parser().parse_args()
     validate_arguments(parsed_args)
     arguments = vars(parsed_args)
+    print(arguments)
```

Comment:
Not a blocking comment, but let's try to be more mindful about these kind of debugging statements.  Eventually these will get written to Stackdriver (or some equivalent service) and we're going to want to be more consistent in how/what we're logging.

Reply:
Addressed in f708656